File: samplejava/src/main/java/pyxis/uzuki/live/richutils/samplejava/BitmapActivity.java
package pyxis.uzuki.live.richutils.samplejava;

import android.graphics.Bitmap;
import android.graphics.drawable.Drawable;
import android.os.Bundle;
import android.support.annotation.Nullable;
import android.widget.ImageView;

import butterknife.BindView;
import kotlin.Unit;
import pyxis.uzuki.live.richutilskt.utils.RichUtils;

/**
 * Created by pyxis on 2017. 6. 12..
 */

public class BitmapActivity extends BaseActivity {
    Bitmap bitmap = null;
    @BindView(R.id.imgView) ImageView imgView;

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_bitmap);

        RichUtils.runAsync(() -> {
            bitmap = RichUtils.downloadBitmap("http://images.goodsmile.info/cgm/images/product/20160606/5715/39417/large/f75b5722baec63a2922a81c3c3ca8743.jpg");
            runOnUiThread(this::init);
            return Unit.INSTANCE;
        });
    }

    private void init() {
        if (bitmap == null)
            return;

        Drawable drawable = RichUtils.bitmapToDrawable(this, bitmap);
        String file = RichUtils.saveBitmapToFile(this, bitmap);

        imgView.setImageBitmap(bitmap);
    }
}

File: sample/src/main/java/pyxis/uzuki/live/richutilssample/BitmapActivity.kt
package pyxis.uzuki.live.richutilssample

import android.graphics.Bitmap
import android.graphics.drawable.Drawable
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_bitmap.*
import pyxis.uzuki.live.richutilskt.utils.bitmapToDrawable
import pyxis.uzuki.live.richutilskt.utils.downloadBitmap
import pyxis.uzuki.live.richutilskt.utils.runAsync
import pyxis.uzuki.live.richutilskt.utils.saveBitmapToFile

class BitmapActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_bitmap)

        var bitmap: Bitmap?
        runAsync {
            bitmap = downloadBitmap("http://images.goodsmile.info/cgm/images/product/20160606/5715/39417/large/f75b5722baec63a2922a81c3c3ca8743.jpg")
            runOnUiThread { init(bitmap) }
        }
    }

    private fun init(bitmap: Bitmap?) {
        if (bitmap == null)
            return

        var drawable: Drawable? = bitmapToDrawable(bitmap)
        // it will be compile error cause drawable is Drawable?
        // var bitmap2 = drawableToBitmap(drawable = drawable)

        var file = saveBitmapToFile(bitmap = bitmap)

        imgView.setImageBitmap(bitmap)
    }
}

